{
  "IssueID": 0,
  "VulnerabilityID": "CVE-2020-24584",
  "Release": "3.11",
  "Package": "py3-django",
  "Repository": "main",
  "FixedVersion": "1.11.29-r1",
  "Subject": "",
  "Description": ""
}